package.json

```diff
@@ -26,6 +26,7 @@
     "url": "http://github.com/Adphorus/react-date-range/issues"
   },
   "dependencies": {
+    "react": "~15",
```

Comment:
Maybe this should better be defined as a peer-dependency?

(And only installed as devDependency here to test the project)